public/video-ui/src/slices/video.ts

```diff
@@ -20,6 +20,9 @@ const video = createSlice({
       state.video = { ...blankVideoData, ...payload };
     },
     setPublishedVideo: (state, { payload }: PayloadAction<Video>) => {
+      state.publishedVideo = { ...blankVideoData, ...payload };
```

Comment:
This is now only used by:

 https://github.com/guardian/media-atom-maker/blob/dba89c1357da55018a51b215262ba03b7e6a30a9/public/video-ui/src/actions/VideoActions/getPublishedVideo.ts#L9-L11.